cmake/dependencies/antares-solver.cmake

```diff
@@ -0,0 +1,64 @@
+#####################
+##  Antares solver  ##
+#####################
+message("DEPENDENCY ANTARES " ${BUILD_antares_solver})
+message("DEPENDENCY ANTARES " ${DOWNLOAD_antares_solver})
+if(BUILD_antares_solver OR DOWNLOAD_antares_solver)
+    find_package(antares-solver QUIET)
+    if (NOT antares-solver_FOUND)
+        if (DOWNLOAD_antares_solver)
+            message(${DEPS_INSTALL_DIR}/..)
+            if (MSVC)
+            message(STATUS "Will download Antares solver windows assets")
+                set(PLATFORM_TYPE installer-64bits.zip)
+            else()
+                execute_process(COMMAND bash "-c" "lsb_release -d | awk -F\"\t\" '{print $2}'" OUTPUT_VARIABLE SYSTEM_FLAVOUR)
+                string(FIND ${SYSTEM_FLAVOUR} "Ubuntu" UBUNTU_FOUND)
+                if (UBUNTU_FOUND EQUAL -1)
+                    message(STATUS "Will download Antares solver CentOS assets")
+                    set(PLATFORM_TYPE CentOS-7.9.2009-ortools-xpress.tar.gz)
```

Comment:
Careful, this asset requires XPRESS. By using it, your Xpansion build will also depend on it.